src/neuroconv/tools/data_transfers/__init__.py

```diff
@@ -1,14 +1,6 @@
 """Collection of helper functions for assessing and performing automated data transfers."""
 
-from ._aws import estimate_s3_conversion_cost
 from ._dandi import automatic_dandi_upload
 from ._globus import get_globus_dataset_content_sizes, transfer_globus_content
-from ._helpers import estimate_total_conversion_runtime
```

Comment:
TBH these functions were probably never used by anyone so OK with removing them. Haven't even reviewed if they are still correct or not